src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.html

```diff
@@ -148,4 +148,37 @@
       </div>
     </ng-container>
   </alg-section>
+
+  <alg-section icon="fa fa-users" i18n-label label="Team" *ngIf="enableTeamSection">
```

Comment:
I tried to remain consistent, but I find the management of enableParticipation to be awkward. To me either a getter is written to declare the logic at one place to be dry, either the check is inlined everywhere which in my opinion is not ideal because then if the condition changes you have to replicate the change everywhere it's inlined.
That was for the same reason - DRY - that I put it as as an arg to its child, I thought that the same check should be used everywhere instead of replicating the logic in the child component.

As for the performance issue due to the change detection system of Angular, that part I know way less. It's strange to me that it would cause more computation. The getter will be run every time it is called yes, but the inline statements will also be run everywhere they are written. Maybe I'm missing something, I don't know… Angular caches in some way the ngIf statements ?